src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -62,7 +65,9 @@ func (r *StatefulSetResource) Ensure(ctx context.Context) error {
 	}
 
 	if errors.IsNotFound(err) {
+		logger.Info(fmt.Sprintf("StatefulSet %s does not exist, going to create one", r.Key().Name))
 		obj, err := r.Obj()
+
```

Comment:
Is this empty line necessary? I would go with empty line between logger and `r.Obj()` invocation.

Reply:
yes, ask the linter :)

Reply:
I'm used to more to 
```
logger.Info(fmt.Sprintf("StatefulSet %s does not exist, going to create one", r.Key().Name))

obj, err := r.Obj()
if err != nil {
```

Reply:
This new line is unrelated to PR

Reply:
oh I see, yeah, I like that more as well

Reply:
fixed